### PR TITLE
[ADD] hr_holidays_working_time module

### PR DESCRIPTION
**Pull request by adrienpeiffer**
- [x] Depends on #93 
- [ ] Depends on https://github.com/odoo/odoo/pull/14584

### CODE REVIEW

**Comment by feketemihai** [comment]
@adrienpeiffer @sbidoul Can you double check if the PR from OCA/hr is suitable for you...https://github.com/OCA/hr/pull/210, it calculated leave days taking in consideration public holidays from https://github.com/OCA/hr/pull/209, employee scheduler and if it doesn't have a scheduler is checking for week days...

File: hr_holidays_working_time/models/hr_holidays.py [hunk 1]
```diff
@@ -55,7 +55,8 @@ def _auto_init(self, cr, context=None):
     number_of_hours = fields.Float(
         'Number of Hours', compute='_compute_number_of_hours', store=True)
     number_of_days_temp = fields.Float(
-        compute='_compute_number_of_hours_from_hours', store=True)
+        compute='_compute_number_of_hours_from_hours', store=True,
+        required=False)
```
Comment (adrienpeiffer) [inline hr_holidays_working_time/models/hr_holidays.py:59]: So, it's an Odoo bug ?

BTW, We have to be sure that there is a value in this field. So, it's necessary to add a constraint.

Comment (JonathanNEMRY) [inline hr_holidays_working_time/models/hr_holidays.py:59]: after checking some Odoo code it appears that actually required attribute is  quite useless into computed field but as it is a mutation from a Float to a computed the required attribute is conserved from the native field declaration. So I don't think it is an Odoo bug. Concerning the constraint is it really necessary as the computing result can not be driven into a False computed value by the added algorithm?

**Comment by JonathanNEMRY** [comment]
till wait merge of #93 to be rebased

File: hr_holidays_working_time/models/hr_holidays.py (hunk 1 above)
Comment (adrienpeiffer) [inline hr_holidays_working_time/models/hr_holidays.py:59]: So, the field on the view is display as required field ?

> Concerning the constraint is it really necessary as the computing result can not be driven into a False computed value by the added algorithm?

Ok, so it's not necessary

**Comment by sbidoul** [comment]
Closing old stuff.

**Pull request closed without merging**